features/attach_invalid_token.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Command behaviour when attach a machine to an Ubuntu Advantage subscription
+
+    Scenario: Attach command in a trusty lxd container
```

Comment:
This scenario is specifically testing attaching with an invalid token, can we update this wording to be explicit about that?

Reply:
My idea for that is:  
- Generating a random token when testing the output if you try an invalid token 
- Providing the token via CLI for the next tests: when you attach to an UA subscription using a valid token.

Reply:
I think Dan's comment comment leads to the following couple questions:
* can this feature file be renamed attach_token.feature. Each feature file in my mind is related to a single 'CLI operation' such as (attach_token.feature, status.feature, enable.feature, disasble.feature, detach.feature etc). This way we have a cohesive set of scenario tests for each CLI command parameter (attach, detach, status, enable, disable) which exercise both "happy path" (correct) and "sad path" (expected failure paths).
* maybe rename  this feature to "Feature: Attach behavior with various tokens"
* As you extend the scenarios we probably need a more correct Scenario text, like "Attach command using invalid token"
* assuming with your next branch you add something like a "Scenario: Attach with a valid contract token in a trusty lxd  container" to this feature file.

Reply:
Oh, this is my mistake actually, I hadn't grokked the file/feature naming. I don't feel strongly that we need a single file per CLI command, though that may be easier to manage until we find that one is getting too large.

Reply:
That last comment was a little terse, let me unpack that a bit: I think leaving the naming as-is for this PR is fine. Then, in the next PR, we can discuss whether or not we want a separate file for the new scenario being added. If we decide that we don't want separate files, then we can rename things as part of that. How does that sound?

Reply:
Works for me. file renaming can be resolved at a later time if needed. Good to know that one feature file per `CLI command` isn't necessarily a goal. So we can drop that first and second bullet point suggestions